openff/interchange/interop/openmm/_import/_import.py

```diff
@@ -348,6 +353,12 @@ def _fill_in_rigid_water_bonds(interchange: "Interchange"):
     from openff.interchange.components.potentials import Potential
     from openff.interchange.models import AngleKey, BondKey, PotentialKey
 
+    if "Bonds" not in interchange.collections:
+        interchange.collections.update({"Bonds": BondCollection()})
+
+    if "Angles" not in interchange.collections:
+        interchange.collections.update({"Angles": AngleCollection()})
+
```

Comment:
In my personal opinion it is a bit more logical to create those empty collections earlier, when all the collections are created. Those collections are anyway created, since `_fill_in_rigid_water_bonds` is anyway called, but now, `_fill_in_rigid_water_bonds` has sort of two tasks, which are not logically connected.

Reply:
I think of this as a sort of special case since it's only hit when the entire system lacks bonds and the collections are created elsewhere in dedicated private methods. I agree it shouldn't make a difference

The only problematic case I can think of is a system that constrained angles but not constrained bonds, which is weird and certainly outside what I think of as a normal treatment of rigid water

Reply:
Ok, I see you point. I think the PR is good. Thanks a lot for a prompt fix.